turbinia/workers/fsstat.py

```diff
@@ -39,13 +39,14 @@ def run(self, evidence, result):
     """
     fsstat_output = os.path.join(self.output_dir, 'fsstat.txt')
 
-    # Since fsstat does not support XFS, we won't run it when we know the
-    # partition is XFS.
-
-    # Note: an evidence object that was serialized will not have a path_spec
-    # because it is removed prior to JSON serialization in evidence.py:243
-    if evidence.type_indicator == "XFS":
-      message = 'Not running fsstat since partition is XFS'
+    if not evidence.path_spec:
```

Comment:
Should we check that this property exists and is not `None` here just to be safe before we reference it later?

Reply:
Done.